pydantic_ai_slim/pydantic_ai/profiles/anthropic.py

```
@@ -282,7 +334,7 @@ def _code_execution_tool_versions(
 ) -> tuple[AnthropicCodeExecutionToolVersion, tuple[AnthropicCodeExecutionToolVersion, ...]]:
     versions: tuple[AnthropicCodeExecutionToolVersion, ...] = ('20250825',)
     default_version: AnthropicCodeExecutionToolVersion = '20250825'
-    if model_name.startswith(_ANTHROPIC_CODE_EXECUTION_20260120_MODEL_PREFIXES):
+    if model_name.startswith(_ANTHROPCIC_CODE_EXECUTION_20260120_MODEL_PREFIXES):
```

Ruff (0.15.21)

[error] Undefined name `_ANTHROPCIC_CODE_EXECUTION_20260120_MODEL_PREFIXES`

(F821)